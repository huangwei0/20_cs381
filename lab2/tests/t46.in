debug off
stopwords off
length 10
load tracks.txt
preprocess
sequence short
sequence float
sequence double
sequence boolean
sequence character
sequence string