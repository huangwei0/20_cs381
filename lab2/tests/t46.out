short of the world is a little girl in my heart
float away from the world is a little girl in my
double trouble in the world is a little girl from me
boolean 
character of the world is a little girl in my heart
string quartet no more than a little girl in the world
